site/src/theme/colorRoles.ts

```diff
@@ -1,49 +1,54 @@
-export type ThemeRole = keyof Roles;
+export type ThemeRole = keyof ColorRoles;
 
 export type InteractiveThemeRole = keyof {
-	[K in keyof Roles as Roles[K] extends InteractiveRole ? K : never]: unknown;
+	[K in keyof ColorRoles as ColorRoles[K] extends InteractiveColorRole
+		? K
+		: never]: unknown;
 };
 
-export interface Roles {
+export interface ColorRoles {
+	/** The default color role for general use */
+	default: ColorRole;
```

Comment:
I'm gonna be defensive about this because I put a lot of thought into the design of this bit of our theme: `default`, the way that you are using it, is not a UI role. As a user, when the app uses the `error` colors I can say "The UI is showing me something is wrong", but I can't say "The UI is showing me a default color role for general use". I think the intent of these roles is really important for consistency throughout the product.

I'm also not convinced that `ColorRole` is a better name. The colors they represent are a consequence, not the primary motivation. Otherwise they'd just be named `red`, `orange`, etc. You're supposed to think about what you're trying to communicate with the color first, and then the colors come from that naturally. Maybe `UiRole` or something if you really think it needs to be changed.

`default` could potentially be an appropriate role name if it was identifying things in the UI as a default target. The default org, the default template version, so on. But there isn't anything about the permissions given to an org role that makes them "default" of that role, especially not when used to highlight all of them.

Reply:
What I want is a way to define styles for a default pill, and although info can work, the name feels like its a special case and not the standard/default color. I would ideally like to keep defaults in palette or someplace outside of roles in case it doesn't fit into background, outline, text, or the fill colors.